install_config/configuring_aws.adoc

```diff
@@ -18,20 +18,52 @@ volumes as persistent storage] for application data. After AWS is configured
 properly, some additional configurations will need to be completed on the
 {product-title} hosts.
 
+[[configuring-aws-permissions]]
+== Permissions
+Configuring AWS for {product-title} requires following permissions on:
+
+* Master:
++
+.AWS permissions required on Master
+[options='header']
+|===
+| Elastic Compute Cloud(EC2) | Elastic Load Balancing
+| _ec2:DescribeVolume_ | _elasticloadbalancing:DescribeTags_
+| _ec2:CreateVolume_ | _elasticloadbalancing:CreateLoadBalancerListeners_
+| _ec2:CreateTags_ | _elasticloadbalancing:ConfigureHealthCheck_
+| _ec2:DescribeInstance_ | _elasticloadbalancing:DeleteLoadBalancerListeners_
+| _ec2:AttachVolume_ | _elasticloadbalancing:RegisterInstancesWithLoadBalancer_
+| _ec2:DetachVolume_ | _elasticloadbalancing:DescribeLoadBalancers_
+| _ec2:DeleteVolume_ | _elasticloadbalancing:CreateLoadBalancer_
+| _ec2:DescribeSubnets_ | _elasticloadbalancing:DeleteLoadBalancer_
+| _ec2:CreateSecurityGroup_ | _elasticloadbalancing:ModifyLoadBalancerAttributes_
+| _ec2:DescribeSecurityGroups_ | _elasticloadbalancing:DescribeLoadBalancerAttributes_
+| _ec2:DescribeRouteTables_ |
+| _ec2:AuthorizeSecurityGroupIngress_ |
+|===
```

Comment:
These are very generic permissions, do we have anything more security restrictive?